packages/bytecode-utils/src/lib/bytecode.ts

```diff
@@ -77,18 +68,46 @@ export const decode = (bytecode: string): DecodedObject => {
 };
 
 /**
- * Get contract's bytecode given an address and an rpc
- * @param address - contract's address
- * @param provider - provider used to get the bytecode
- * @returns bytecode
+ * Split bytecode into execution and auxdata
+ * @param bytecode - hex of the bytecode with 0x prefix
+ * @returns array first element is execution bytecode, second element is auxdata
  */
-export const get = async (
-  address: string,
-  provider: EthereumProvider
-): Promise<string> => {
-  const bytecode = await provider.request({
-    method: 'eth_getCode',
-    params: [address, 'latest'],
-  });
-  return bytecode as string;
+export const splitAuxdata = (bytecode: string): string[] => {
+  if (bytecode.length === 0) {
+    throw Error('Bytecode cannot be null');
+  }
+  if (bytecode.substring(0, 2) !== '0x') {
+    throw Error('Bytecode should start with 0x');
```

Comment:
Why not just handle bytecodes without 0x?
```suggestion
    bytecode = "0x" + bytecode
```